lib/active_set/attribute_instruction.rb

```diff
@@ -1,16 +1,20 @@
 # frozen_string_literal: true
 
-require 'active_support/core_ext/array/wrap'
-
 class ActiveSet
-  class Instruction
+  class AttributeInstruction
+    attr_accessor :processed
```

Comment:
This allows an Operation to signal when some particular Strategy has successfully processed an attribute instruction, which the Operation can then use to be sure to not re-process that instruction.